object/attribute.go

```diff
@@ -41,6 +41,17 @@ func NewAttribute() *Attribute {
 	return NewAttributeFromV2(new(object.Attribute))
 }
 
+// NewParameterizedAttribute creates and initializes new [Attribute].
+//
+// Works similar as [NewAttribute] but allows to set attribute key and value in place.
+func NewParameterizedAttribute(key, value string) *Attribute {
```

Comment:
maybe we do not need `func NewAttribute() *Attribute`? i think it will always have just key and value so nothing to init via a constructor

Reply:
I agree with you, but we may leave it for back compatibility. We may mark it as obsolete and remove in the next release

Reply:
> We may mark it as obsolete and remove in the next release

sure. just do not like `NewParameterizedAttribute` name. and `NewAttribute` is the only reason it is named so

Reply:
Updated